src/store/reducers/describe.ts

```diff
@@ -28,21 +27,34 @@ const describe: Reducer<
             };
         }
         case FETCH_DESCRIBE.SUCCESS: {
-            let newData;
+            const data = action.data;
 
-            if (action.data.Path) {
+            const isCurrentDescribePath = data.Path === state.currentDescribePath;
```

Comment:
May be it will be more clear if we check:
```
if (!isCurrentDescribePath) {
return {...state}
}
```

Reply:
Thanks, it is much better